packages/fxa-shared/test/metrics/navigation-timing.ts

```diff
@@ -3,25 +3,14 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import sinon from 'sinon';
+import { observeNavigationTiming } from '../../metrics/navigation-timing';
 
 type MockEntryList = { getEntries: () => [object] };
 type ObsCallback = (_entries: MockEntryList, _obs: object) => undefined;
-type sinonStub = ReturnType<typeof sinon.stub>;
 let cb = (_entries: MockEntryList, _obs: object) => {};
 
 const sandbox = sinon.createSandbox();
-(global.navigator as unknown) = { sendBeacon: sandbox.stub() };
```

Comment:
`navigator.sendBeacon` is the only hangup. it seems esbuild-register handles import caching differently enough from ts-node that this stub doesn't apply to the navigator in navigation-timing.ts. `sendFn` is simple enough that I don't think we need to assert the arguments passed to sendBeacon.